contrib/vcloud/build.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is quite a lot of stuff just to basically make a single call to Ivy, which should be doable with one command, right? Furthermore, it adds the dependency to `ant` to something that would otherwise require only Python and Java. Can't we get rid of this?